code/modules/mob/living/carbon/xenomorph/xenomorph.dm

```diff
@@ -221,8 +221,9 @@
 	if(ishuman(L))
 		if(!chestburst)
 			do_attack_animation(L, ATTACK_EFFECT_GRAB)
-			if(!do_mob(src, L , XENO_PULL_CHARGE_TIME, BUSY_ICON_HOSTILE))
-				return FALSE
+			if(!get_filter("warrior_lunge")) //Warriors don't care about drag delay; they're MMA champions
```

Comment:
I'd prefer if this checked if the mob is conscious instead of checking for filter